addons/mail/static/src/core/common/suggestion_hook.js

```diff
@@ -182,6 +194,23 @@ export class UseSuggestion {
         this.composer.selection.end = before.length + option.label.length + 1;
         this.composer.forceCursorMove = true;
     }
+    insertHtml(option) {
+        const { startContainer, endContainer, endOffset } =
+            this.comp.editor.shared.selection.getEditableSelection();
+        const range = {
+            startContainer,
+            startOffset: this.search.position,
+            endContainer,
+            endOffset,
+        };
+        this.comp.editor.shared.delete.deleteRange(range);
+        if (option.cannedResponse) {
+            this.composer.cannedResponses.push(option.cannedResponse);
+        }
+        this.comp.editor.shared.dom.insert(option.label);
+        this.comp.editor.shared.dom.insert("\u00A0");
```

Comment:
This could be simple space but maybe just remove it instead, this likely won't be useful